File: PHP/Program/Clothing.php
<!-- /* 
    Saya Tattha Maharany Yasmin Akbar dengan NIM 2201805 mengerjakan soal Latihan 2 
    dalam Praktikum mata kuliah Desain dan Pemrograman Berbasis Objek, untuk keberkahan-Nya
    maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamin.
*/ -->
<?php

require_once('Shirt.php'); // Jika file Shirt.php berada dalam direktori yang sama

class Clothing extends Shirt {
    // attributes
    private $size = "";
    private $material = "";
    private $gender = '';

    // constructor
    public function __construct($size = "", $material = "", $gender = '') {
        parent::__construct(); // Memanggil konstruktor kelas induk (Clothing)
        $this->size = $size;
        $this->material = $material;
        $this->gender = $gender;
    }

    // getter dan setter
    public function get_size() {
        return $this->size;
    }

    public function set_size($size) {
        $this->size = $size;
    }

    public function get_material() {
        return $this->material;
    }

    public function set_material($material) {
        $this->material = $material;
    }

    public function get_gender() {
        return $this->gender;
    }

    public function set_gender($gender) {
        $this->gender = $gender;
    }

    // destructor (opsional, tidak perlu didefinisikan dalam PHP)
    // public function __destruct() {
    //     // kode pembersihan
    // }
}

?>

File: PYTHON/Program/Clothing.py
    # Saya Tattha Maharany Yasmin Akbar dengan NIM 2201805 mengerjakan soal Latihan 2 
    # dalam Praktikum mata kuliah Desain dan Pemrograman Berbasis Objek, untuk keberkahan-Nya
    # maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamin.
    
from Shirt import Shirt
# atribut
class Clothing(Shirt):
    __size = ""
    __material = ""
    __gender = ''
    
    # konstruktor
    def __init__(this, size = "", material = "", gender = ''):
        this.__size = size
        this.__material = material
        this.__gender = gender
    
    #  getter setter
    def get_size(this): return this.__size
    def set_size(this, size): this.__size = size
    
    def get_material(this): return this.__material
    def set_material(this, material): this.__material = material
    
    def get_gender(this): return this.__gender
    def set_gender(this, gender): this.__gender = gender